bin/get_schemas.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import requests
+import configparser
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+# prsvtoken.py needs to be in the same directory for this to work
+import prsvtoken
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--instance',
+        '-i',
+        type=str,
+        required=True,
+        choices=['test', 'prod'],
```

Comment:
I added the choices arg here. It can be very handy. Automatically tests if the input is in the list.